docs/pages/trustedclusters.mdx

```diff
@@ -274,7 +274,7 @@ spec:
       'environment': 'production'
 ```
 
-Now, we need to establish trust between roles "root:admin" and "leaf:admin". This is
+Now, we need to establish trust between roles "root:admin" and "leaf:access". This is
```

Comment:
Great catch!

I'm wondering if we want to elevate someone's access here (even if this is just an example). I'd think we'd want the level of access to be the same. Especially from a leaf cluster to a root cluster.

I'd think we'd want to update the YAML to have the same mapping instead (which seems to be the intent of the comment too):

```yaml
role_map:
    - remote: admin
      # admin <-> admin works for the Open Source Edition. Enterprise users
      # have great control over RBAC.
      local: [admin]
```

My guess is that this snippet was copy-pasted in from somewhere.